modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/PublicationTimeline.js

```diff
@@ -31,7 +31,10 @@ const PublicationTimeline = ({namespace, timelineItemsURL}) => {
 			{
 				ctCollectionId,
 				mvcRenderCommandName,
-				p_p_id: namespace,
+				p_p_id:
+					namespace.startsWith('_') && namespace.endsWith('_')
+						? namespace.slice(1, namespace.length - 1)
+						: namespace,
```

Comment:
There's a util to get the portletId from the portletNamespace: [getPortletId](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/get_portlet_id.js).

You can use `Liferay.Util.getPortletId` or import it from `frontend-js-web`.